src/gui.cpp

```diff
@@ -0,0 +1,36 @@
+#include "gui.h"
+
+void GUI::clear_screen() {
+  // Mac/Linux
+#ifndef DEBUG
+  system("clear");
+#endif
+}
+
+void GUI::print_border() {
+  char border[BOARD_WIDTH + 1] = "|";
+  for (int i = 1; i < BOARD_WIDTH - 1; ++i) {
+    border[i] = '=';
+  }
+  border[BOARD_WIDTH - 1] = '|';
+  printf("%s\n", border);
+}
+
+void GUI::print_item (const char* item, bool right_justify) {
+  if (right_justify) {
+    printf("| %*s |\n", INNER_WIDTH - INNER_PAD, item);
+  } else {
+    printf("| %-*s |\n", INNER_WIDTH - INNER_PAD, item);
+  }
+}
+
+void GUI::print_header(const char* title, bool clear) {
```

Comment:
idc about doing it mostly C-style but the main thing I would want is `std::string`